hypershift/requirements.txt

```diff
@@ -1,2 +1,3 @@
 git-python
 ConfigParser
+elasticsearch
```

Comment:
this is not required, ES module is imported on libs/requirements.txt

Reply:
Update